cel/env.go

```diff
@@ -740,6 +757,90 @@ func getStdEnv() (*Env, error) {
 	return stdEnv, stdEnvErr
 }
 
+// interopCELTypeProvider layers support for the types.Provider interface on top of a ref.TypeProvider.
+type interopCELTypeProvider struct {
+	ref.TypeProvider
+}
+
+// FindStructType returns a types.Type instance for the given fully-qualified typeName if one exists.
+//
+// This method proxies to the underyling ref.TypeProvider's FindType method and converts protobuf type
+// into a native type representation. If the conversion fails, the type is listed as not found.
+func (p *interopCELTypeProvider) FindStructType(typeName string) (*types.Type, bool) {
+	if et, found := p.FindType(typeName); found {
+		t, err := types.ExprTypeToType(et)
+		if err != nil {
+			return nil, false
```

Comment:
Swallowing the error feels...bad. Should the interface allow returning error? Maybe instead of `bool` we use `error` and a singleton `NotFoundErr` or something to communicate it? Or use 3 return values?

Not a requirement, just a thought.

Reply:
In this case, the not found result is usually surfaced as an error by something higher up, like the type-checker. Ideally, these conversion steps are needed in the long term, but they're definitely needed for backwards compatibility in the short-term.

I don't like swallowing the error, but the behavior is likely the same in either case: a misconfigured type doesn't exist, act accordingly. I felt this was better than a panic, but could see the argument for it.